servers/playmides/metadata.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "playmides",
+  "name": "PlayMides Network",
+  "description": "We're a competitive mini games & pvp 1.8.8 network",
+  "addresses": [
+    "playmides.net"
+  ],
+  "primaryAddress": "playmides.net",
+  "minecraftVersions": [
```

Comment:
only put Minecraft versions that lunar client supports.

Reply:
Fixed!